web/thesauruses/lua/5/strings.json

```
@@ -0,0 +1,272 @@
+{
+    "meta": {
+      "language": "lua",
+      "language_name": "Lua",
+      "structure": "strings",
+      "language_version": "5.3.6"
+    },
+    "concepts": {
+      "is_primitive_or_not": {
+        "name": "Is this a built-in type in this language?",
+        "code": "Yes"
+      },
+      "import": {
+        "name": "Import the string class",
+        "code": "",
+        "comment": "Does not require to import as it is in builtins"
+      },
+      "default_string_byte_encoding": {
+        "name": "Default byte encoding (ex: ASCII UTF-8, UTF-16, etc.)",
+        "not-implemented": true
+      },
+      "create_new_string": {
+        "name": "Create new string",
+        "code": "str1 = 'New string Example1' \nstr2 = \"New string Exampl2\""
+      },
+  
+      "create_multiline_string": {
+        "name": "Create new multi-line string",
+        "code": "[['Multiline string']]"
+      },
+      "assign_new_string": {
+        "name": "Assign string from another string",
+        "code": "a = 'a'\na = 'b'"
+      },
+      "destroy_string": {
+        "name": "Destroy string",
+        "not-implemented": "true"
+      },
+      "length_of_string": {
+        "name": "Length of string",
+        "code": "string.len('string')"

```

Probably add the method syntax
```suggestion
        "code": [
            "'string':len()",
            "string.len('string')"
        ]

```

---

Perhaps to make it less confusing, do something like `'my text':len()` and `string.len('my text')` to help distinguish string (a variable) from string (text) vs string (a class)